### ISSUE TITLE
Does not work with Gradle 5.3

### ISSUE THREAD

**Issue by henrik242** [psxpaul/gradle-execfork-plugin#18]
```
   > Could not create task of type 'JavaExecFork'.
      > org.gradle.process.internal.DefaultJavaForkOptions: method <init>(Lorg/gradle/internal/file/PathToFileResolver;)V not found
```

**Comment by henrik242** [comment]
@psxpaul Do you know if there is an easy fix for this?

**Comment by psxpaul** [comment]
This should be fixed in 0.1.10
